ftsbot/cogs/textcommands.py

```diff
@@ -427,21 +427,30 @@ async def on_liquipedia_error(
 	)
 	@app_commands.describe(
 		wiki='Which wiki do you want the notability guidelines for?',
+		user='Which user is the guidelines directed towards?'
 	)
 	@app_commands.autocomplete(
 		wiki=autocomplete.wiki
 	)
 	async def notability(
 		self,
 		interaction: discord.Interaction,
-		wiki: typing.Optional[str]
+		wiki: typing.Optional[str],
+		user: typing.Optional[discord.User]
 	):
 		usewiki = None
 		if wiki in data.wikis:
 			usewiki = wiki
 		elif isinstance(interaction.channel, discord.channel.TextChannel) and interaction.channel.name in data.wikis:
 			usewiki = interaction.channel.name
-		if usewiki is not None:
+		if usewiki is None:
+			await interaction.response.send_message(
+				embed=discord.Embed(
+					colour=discord.Colour(0xff0000),
+					description='No wiki specified'
+				)
+			)
+		if user is None:
```

Comment:
imo `elif` here

Reply:
Could also just be an `else`, no?

Also error messages should be ephemeral like 
```await interaction.response.send_message(embed=discord.Embed(...), ephemeral=True)```

Reply:
there is an else below

atm it is

if
if
else

i suggest
if
elif
else

so we do not get 2 messages in some cases

Reply:
oh I thought you wanted 
```
if wiki in data.wikis:
			usewiki = wiki
		elif isinstance(interaction.channel, discord.channel.TextChannel) and interaction.channel.name in data.wikis:
			usewiki = interaction.channel.name
		else:
			await interaction.response.send_message(
				embed=discord.Embed(
					colour=discord.Colour(0xff0000),
					description='No wiki specified'
				)
			)
```

Reply:
basically this would catch usewiki not being set right where it's set

Reply:
so I wanted to roll it into the previous if/elif and you wanted to roll it into the later if/elif

Reply:
my point is that
```
if usewiki is None:
	send message
if user is None:
	send message
else
	send message
```

will (try to) send 2 messages if usewiki is None
(and produce a bs link in the 2nd message)

while
```
if usewiki is None:
	send message
elif user is None:
	send message
else
	send message
```
doesn't have that issue

alternative of couse would be
```
if usewiki is None:
	send message
else
	if user is None:
		send message
	else
		send message
```

Reply:
Resolved conditionals by performing early returns, lmk if that's not preferred

Reply:
This is fine